README.md

```diff
@@ -36,6 +36,24 @@ to your Github account:
 4. Under "Subscriptions.. Participating, @mentions and custom", ensure that you have "Email" ticked
 
 You should now receive an email whenever the changelog is updated.
+### 28 June 2023
+
+The following changes were deployed into sandbox:
+
+#### cis-deductions-api
+
+New API Version `v2.0`
+Updated Endpoint: `Retrieve Cis Deductions for Subcontractor`
+
+* `API#1572`-`Retrieve Cis Deductions for Subcontractor`:
```

Comment:
I think we could actually make this all into one:

```
The following changes were deployed into sandbox:

#### cis-deductions-api

New API Version `v2.0`
Updated Endpoint: `Retrieve Cis Deductions for Subcontractor`

* Added `taxYear` and `source` path parameters
* Added `TAX_YEAR_RANGE_INVALID` gov-test-scenarios
* Removed `DATE_RANGE_OUT_OF_DATE` gov-test-scenario

```

And also add in about how the old QPs were removed

Reply:
We could potentially just explicitly say something along the lines of "updated endpoint URL"

Reply:
If we make it into one, then it doesn't highlight exactly what changes were made for each API. The GT "DATE_RANGE_OUT_OF_DATE" only applies to API#1572.